ui/pages/swaps/searchable-item-list/__snapshots__/searchable-item-list.test.js.snap

```diff
@@ -3,7 +3,6 @@
 exports[`SearchableItemList renders the component with initial props 1`] = `
 <div
   class="MuiFormControl-root MuiTextField-root searchable-item-list__search MuiFormControl-fullWidth"
-  data-testid="search-list-items"
```

Comment:
Previously the `data-testid` prop was being passed through by our component to the wrapper div of the Material UI Text Field component. Moving it to the input component results in this fixture changing to reflect that.

This has no impact on any tests, because this `search-list-items` ID was never referenced anywhere. I do not know why it was added.